docs/deploy.md

```diff
@@ -1,26 +1,28 @@
-# Deploying and saving a model
+# Making Predictions and Deploying Models
```

Comment:
This is intentional. Though you are right in that I skipped the `.save()` step! Great!